README.md

```diff
@@ -487,6 +487,17 @@ The TypedData class is a wrapper class for the serialized binary data. It presen
 
 TypedData does late deserialization of the data only when the get method is called.
 
+This TypedData allows you to retrieve the data type of the underlying binary to be used when being deserialized. This class represents the type of a Hazelcast serializable object. The fields can take the following values:
+1. <b>Primitive types</b>: factoryId=-1, classId=-1, typeId is the type id for that primitive as listed in
+@link SerializationConstants
+2. <b>Array of primitives</b>: factoryId=-1, classId=-1, typeId is the type id for that array as listed in
+@link SerializationConstants
+3. <b>IdentifiedDataSerializable</b>: factory, class and type ids are non-negative values as registered by
+the DataSerializableFactory.
+4. <b>Portable</b>: factory, class and type ids are non-negative values as registered by the PortableFactory.
+5. <b>Custom serialized objects</b>: factoryId=-1, classId=-1, typeId is the non-negative type id as 
```

Comment:
....the non-negative type ID as....

Reply:
and please make "id"s in the code comments below "ID".

Reply:
OK, corrected